Makefile

```diff
@@ -84,10 +84,18 @@ debug: .state/docker-build
 	docker-compose run --rm --service-ports web
 
 tests: .state/docker-build
+	docker-compose up -d db
```

Comment:
This one is a freebie. I can make it in its own PR but I think it's ok to have it here.